server/src/services/cameraService.ts

```diff
@@ -84,8 +93,19 @@ export class CameraService {
     const resp = {} as PhotoData;
 
     try {
+      /* 
+      // this is for prototype 1 
       resp.apiResponse = await sendToApi(photo);
       resp.encodedImg = encodedImg;
+      */
```

Comment:
if possible could you remove the prototype 2 code from here? and uncomment the prototype 1 code